pkg/cloudevents/datacodec/codec_test.go

```diff
@@ -30,6 +32,11 @@ func TestCodecDecode(t *testing.T) {
 			wantErr:     `[decode] unsupported content type: "unit/testing-invalid"`,
 		},
 
+		"text/plain": {
+			contentType: "text/plain",
+			in:          "hello😀",
```

Comment:
no cat emoji ? :cat:

Reply:
Holy moly, there's a whole library of unicode cats:
https://emojipedia.org/cat-face-with-wry-smile/
Somebody has waaay too much time on their hands.